app/containers/CommitUnknownUserAftermathContainer.tsx

```diff
@@ -51,105 +55,133 @@ const estTextFieldStyles = {
+export class CommitUnknownUserAftermath extends React.Component<
+  InjectedFormProps<ICommitFundsUnknownUserAftermathForm> & ICommitFundsUnknownUserAftermathProps
+> {
+  public componentDidMount() {
+    if (this.props.initialValues && this.props.initialValues.address) {
+      (this.props.handleSubmit as any)();
+    }
+  }
 
-      <div className={styles.infoBox}>
-        <h4>Your wallet address:</h4>
-        <div className={styles.value}>
-          <form
-            onChange={() => {
-              // tslint:disable-next-line:jsx-no-lambda
-              // say thank you to redux-form. Is there a better way to achieve auto submitting? what if async validation is required?
-              // using global onChange didnt work (and still there is no way to check validation status)
-              setTimeout(handleSubmit);
-            }}
-          >
-            <Field
-              name="address"
-              component={TextField}
-              validate={ethereumAddressValidator}
-              props={
-                {
-                  inputStyle: estTextFieldStyles.inputStyle,
-                  style: estTextFieldStyles.style,
-                } as any
-              }
-            />
-          </form>
-        </div>
-      </div>
+  public render() {
+    const {
+      lockedAmountEth,
+      unlockDateEth,
+      neumarkBalanceEth,
+      lockedAmountEur,
+      unlockDateEur,
+      neumarkBalanceEur,
+      handleSubmit,
+      getTokenHolderAgreementTags,
+      getReservationAgreementTags,
+      reservationAgreementHash,
+      tokenHolderAgreementHash,
+      neumarkBalance,
+      isAddressSet,
+      address,
+    } = this.props;
 
-      {isAddressSet &&
+    const showEther = parseFloat(lockedAmountEth) > 0;
+    const showEuro = parseFloat(lockedAmountEur) > 0;
+
+    return (
+      <div className={styles.aftermath}>
         <div>
-          <div className={styles.infoBox}>
-            <h4>Your NEU balance: </h4>
-            <div className={styles.value}>
-              <MoneyComponent value={neumarkBalance} tokenType={TokenType.NEU} />
-            </div>
+          <div className={styles.header}>
+            This is a sneak peak of your committed funds in the Neufund ICBM.
           </div>
+        </div>
 
-          <div className={styles.infoBox}>
-            <div className={styles.value}>
-              <DownloadDocumentLink
-                key="neumark_token_holder_agreement"
-                documentHash={tokenHolderAgreementHash}
-                getTags={getTokenHolderAgreementTags}
-                outputFilename="neumark_token_holder_agreement"
-              >
-                Token Holder Agreement
-              </DownloadDocumentLink>
-            </div>
+        <div className={styles.infoBox}>
+          <h4>Your wallet address:</h4>
+          <div className={styles.value}>
+            <form
+              onSubmit={e => {
+                // tslint:disable-next-line:jsx-no-lambda
```

Comment:
since we are ignoring `jsx-no-lambda` many times, why not ignore it in global file?

Reply:
yeah i also have this thought xD i am gonna do this

Reply:
but in general, we should care... thing is that here we don't care about performance since its just a couple of components anyway